docs/WindowsBuild.md

```diff
@@ -41,6 +41,8 @@ it provides some of the needed headers and libraries.
 1. Clone `apple/swift-corelibs-foundation` into a folder named `swift-corelibs-foundation`
 1. Clone `apple/swift-corelibs-xctest` into a folder name `swift-corelibs-xctest`
 1. Clone `apple/swift-lldb` into a folder named `lldb`
+1. Clone `apple/swift-llbuild` into a folder named `swift-llbuild`
+1. Clone `apple/swift-package-manager` into a folder named `swift-package-manager`
```

Comment:
Not sure, if this true on Windows but on macOS it's configured to swiftpm.

Reply:
AFAIK the only repos for which the name actually _matters_ are `llvm`, `clang`, and `compiler-rt`; the rest just need to be used consistently, at least as things currently stand.

Reply:
As I've been working on the s-p-m port, I've learnt that `llbuild` is also an odd apple that forks.